Tests/Functional/Domain/Service/RestrictionIdentifierFabricTest.php

```diff
@@ -55,14 +65,12 @@ class RestrictionIdentifierFabricTest extends UnitTestCase
      */
     public function setUp()
     {
-        $this->configuration = $this->getMock(
-            'Aoe\FeloginBruteforceProtection\System\Configuration',
-            array(),
-            array(),
-            '',
-            false
-        );
-        $this->frontendUserAuthentication = $this->getMock('TYPO3\CMS\Frontend\Authentication\FrontendUserAuthentication');
+        $this->configuration = $this->getMockBuilder('Aoe\FeloginBruteforceProtection\System\Configuration')
```

Comment:
There is a nice and short solution for simple mocks:
Just use `$this->createMock('Aoe\FeloginBruteforceProtection\System\Configuration')`
It does a couple of things out of the box:

```
$this->getMockBuilder($originalClassName)
                    ->disableOriginalConstructor()
                    ->disableOriginalClone()
                    ->disableArgumentCloning()
                    ->disallowMockingUnknownTypes()
                    ->getMock();```